frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -77,7 +75,11 @@ const FeatureOverviewMetaData = () => {
                             marginRight: theme.spacing(2),
                             height: '40px',
                             width: '40px',
+                            padding: theme.spacing(0.5),
```

Comment:
Wanted to make it styled component, but was not able to because how **IconComponent** is created.

Reply:
You can still refactor it to a separate component if you'd like to.